package.json

```diff
@@ -25,22 +25,23 @@
     "url": "https://github.com/atais/ng2-eonasdan-datetimepicker.git"
   },
   "devDependencies": {
-    "@types/core-js": "^0.9.34",
-    "ts-loader": "^1.2.2",
-    "typescript": "^2.0.10",
+    "@types/core-js": "^0.9.36",
+    "ts-loader": "^2.0.1",
+    "typescript": "^2.1.6",
     "zone.js": "^0.7.2"
   },
   "dependencies": {
-    "@angular/common": "^2.0.0",
-    "@angular/compiler": "^2.0.0",
-    "@angular/core": "^2.0.0",
-    "@angular/platform-browser": "^2.0.0",
-    "@angular/platform-browser-dynamic": "^2.0.0",
-    "bootstrap": "*",
+    "@angular/common": "^2.4.4",
+    "@angular/compiler": "^2.4.4",
+    "@angular/core": "^2.4.4",
+    "@angular/platform-browser": "^2.4.4",
+    "@angular/platform-browser-dynamic": "^2.4.4",
+    "bootstrap": "~3.3.6",
```

Comment:
I would say, overall the deps should be same as in 
https://github.com/Eonasdan/bootstrap-datetimepicker/blob/master/package.json

```
  "dependencies": {
    "bootstrap": "^3.3",
    "jquery": "^1.8.3 || ^2.0 || ^3.0",
    "moment": "^2.10",
    "moment-timezone": "^0.4.0"
  },
``` 
I could be wrong here, though.